launch/ply_importer_demo.launch

```diff
@@ -8,8 +8,12 @@
   <arg name="rotation" default="0 0 0.7071068 0.7071068" />
 
   <node name="ply_to_sdf" pkg="voxblox_ground_truth" type="ply_to_sdf" output="screen"
-        args="$(arg ply_input_file_path) $(arg tsdf_output_file_path) $(arg voxel_size) $(arg scale) $(arg translation) $(arg rotation)"/>
+        args="$(arg ply_input_file_path) $(arg tsdf_output_file_path) $(arg voxel_size) $(arg scale) $(arg translation) $(arg rotation)">
+    <param name="fill_inside" value="true" />
+    <param name="floodfill_unoccupied" value="false" />
+    <param name="tsdf_slice_height" value="4" />
```

Comment:
It'd nice if these settings were also available through the command-line. I can take care of this.
@helenol what is your opinion on using GFlags? That could be a good alternative to the [current simple implementation](https://github.com/ethz-asl/voxblox_ground_truth/blob/master/src/user_interfaces/ply_to_sdf_script.cpp#L25) which assumes a fixed order etc.

Reply:
Yeah for command-line tools for sure. Just depends if you want this to be a ROS util or not (if you commit to ROS, i.e., for visualization anyway, then you can always do _fill_inside:=true from the command-line for example). If a non-ROS tool, gflags are great but they're a pain to use from inside ROS.